app/services/twilio_service.rb

```diff
@@ -30,10 +30,11 @@ def send_unread_message_reminder(url)
   # options[:to] = "+85261111111"
   # options[:body] = "SMS body"
   def send_sms(options)
-    options = {to: @user.mobile}.merge(options)
     if send_to_twilio?
+      options = { to: @user.mobile }.merge(options)
       TwilioJob.perform_later(options)
     else
+      options = { to: @user.mobile.presence || @user.email.presence } .merge(options)
```

Comment:
It should be based on what user is using for login and not based on what is present.

Thoughts @abhinav098 ?

Reply:
Yeah i think you're right.

Reply:
🙂